tailcall-tracker/src/collect/mod.rs

```diff
@@ -0,0 +1,2 @@
+pub mod ga;
+pub mod posthog;
```

Comment:
Don't expose posthog or GA publicly. I would recommend we create a wrapper type that composes to collectors and dispatches the event to both the collectors.